python/requirements/ml/requirements_tune.txt

```diff
@@ -34,6 +34,7 @@ lightning-bolts==0.4.0
 protobuf==3.19.6
 pytorch-lightning==1.6.5
 fairscale==0.4.6
+s3fs==2023.1.0
```

Comment:
General question: How do we determine dependency versions for CI?

Reply:
As recent as possible but compatible with everything else - newer versions weren't found on pypi for Python 3.7 on CI (yet), so I went with the latest available one.